output/stencil/package.json

```
@@ -0,0 +1,48 @@
+{
+  "name": "@db-ui/web-components",
+  "version": "0.0.0",
+  "description": "Stencil components for @db-ui/components",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/db-ui/mono.git"
+  },
+  "type": "module",
+  "license": "Apache-2.0",
+  "main": "./dist/index.js",
+  "module": "./dist/index.js",
+  "exports": {
+    ".": {
+      "types": "./dist/loader/types/index.d.ts",
+      "import": "./dist/loader/index.js"
+    }
+  },
+  "types": "./dist/index.d.ts",
+  "files": [
+    "dist/"
+  ],
+  "scripts": {
+    "build": "stencil build",
+    "build:cem": "cem analyze",
+    "mv:dist": "cpr dist ../../build-outputs/web-components/dist -o",
+    "mv:package.json": "cpr package.json ../../build-outputs/web-components/package.json -o",
+    "mv:readme": "cpr README.md ../../build-outputs/web-components/README.md -o",
+    "postbuild": "npm-run-all -p build:* -p mv:*",
+    "start": "stencil build --watch"
+  },
+  "devDependencies": {
+    "@custom-elements-manifest/analyzer": "^0.10.3",
+    "@stencil/core": "4.21.0",
+    "cem-plugin-expanded-types": "^1.3.2",
+    "custom-element-jet-brains-integration": "^1.6.2",
+    "custom-element-vs-code-integration": "^1.4.1",
+    "workbox-build": "4.3.1"
+  },
+  "publishConfig": {
+    "registry": "https://registry.npmjs.org/",
+    "access": "public"
+  },
+  "web-types": "./dist/web-types.json",
+  "dependencies": {
+    "@custom-elements-manifest/to-markdown": "^0.1.0"
+  }
```

```suggestion
  "web-types": "./dist/web-types.json"
```

I might have missed something, but this dependency doesn't seem to get used within the resulting package, so it should probably get moved from the `dependencies` to the `devDependencies` section.